external/upstream/mdi/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+find_package(mdi CONFIG QUIET)
```

Comment:
probably want to add `0.6.4` (or whatever is the minimal compatible version) here as 2nd argument.

often there's a similar `find_package` section in psi4/CMakeLists.txt. not req'd for pure-py but can help in locating.

Reply:
I've made both changes.